front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -829,10 +829,7 @@ export async function submitAssistantBuilderForm({
         status: isDraft ? "draft" : "active",
         scope: builderState.scope,
         actions: removeNulls([actionParam]),
-        generation: {
-          model: builderState.generationSettings.modelSettings,
-          temperature: builderState.generationSettings.temperature,
-        },
+        generation: {}, // Warning should not be empty otherwise no generation will be done
```

Comment:
So this will never contain anything more, right?

Reply:
It will contain the name & description of the action!

Reply:
👍